box-share-sdk/src/main/res/values/strings.xml

```diff
@@ -1,5 +1,7 @@
 <resources>
     <string name="box_sharesdk_name">Share</string>
+    <string name="box_sharesdk_sample_name">Share SDK Sample</string>
```

Comment:
Please add descriptions for the strings..so, our localization team knows what they mean.